client/components/Coverages/CoverageAddAdvancedModal.tsx

```diff
@@ -407,14 +264,66 @@ class CoverageAddAdvancedModalComponent extends React.Component<IProps, IState>
                                 text={gettext('Save')}
                                 type="primary"
                                 style="filled"
-                                disabled={!this.state.isDirty || !savePermitted}
+                                disabled={!this.state.isDirty || !canSave}
                                 onClick={() => {
                                     this.save();
                                 }}
                             />
                         </Spacer>
                     </Spacer>
-                </Modal.Footer>
+                )}
+            >
+                <Spacer v gap="8" justifyContent="center" alignItems="center">
+                    {this.state.coverages.map((coverage, index) => {
+                        const isActive = coverage.workflow_status === 'active';
+
+                        return (
+                            <div
+                                key={index}
```

Comment:
using loop indexes as keys is not recommended. We should instead use a stable value. Maybe like this it would be a bit better
```suggestion
                                key={coverage.coverage_id ?? index}
```

Reply:
it should be ok here since there's no reordering